Dockerfile.import

```diff
@@ -0,0 +1,12 @@
+FROM debian:stretch
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
```

Comment:
You mean for each container image import and kosmtik? No, there is no way to generalize that as both are completely independent from each other. But the containers are only build if you start them the first time, so this is only done once.

Ideally we would publish both images to the Docker registry so that users only would have to download the pre-build images and not have to build them themselves. But this opens up questions about versioning of the images. Separately or in line with the style? The images will not be updated often.

Also the import should be only done once if you don't change your data. You have to be careful not to type `docker-compose up` but instead `docker-compose up import` or `docker-compose up kosmtik`. The first command without target will start both containers simultaneously.